package.json

```diff
@@ -13,6 +13,7 @@
     "@types/node": "20.3.3",
     "@types/react": "18.2.14",
     "@types/react-dom": "18.2.6",
+    "alphadex-sdk-js": "^0.5.2",
```

Comment:
Can you please update to the latest version of the SDK. I made one change to export the StaticState type. It is needed for some of the changes I propose below.